dvc/repo/experiments/executor/base.py

```diff
@@ -587,9 +587,10 @@ def _repro_dvc(  # noqa: C901
             post_live_metrics(
                 "done",
                 info.baseline_rev,
-                info.name,
+                info.name,  # type: ignore[arg-type]
                 "dvc",
                 experiment_rev=dvc.experiments.scm.get_ref(EXEC_BRANCH),
+                metrics=to_studio_metrics(dvc.metrics.show()),
```

Comment:
Use [`get_in`](https://funcy.readthedocs.io/en/stable/colls.html#get_in)?

```suggestion
                metrics=get_in(dvc.metrics.show(), ["workspace", "data"], default={}),
```